hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/fsck/ContainerHealthTask.java

```diff
@@ -146,7 +155,9 @@ private long processExistingDBRecords(long currentTime) {
             currentContainer = setCurrentContainer(rec.getContainerId());
           }
           if (ContainerHealthRecords
-              .retainOrUpdateRecord(currentContainer, rec)) {
+              .retainOrUpdateRecord(currentContainer, rec) &&
+              !containerDeletedInSCM(currentContainer.getContainer())
```

Comment:
Can we make this SCM call only if the container is seen as MISSING, and not for mis replicated cases?

Reply:
@avijayanhwx Thanks for the review. Updated the patch to only check SCM for missing containers.